openchemistry/_data.py

```diff
@@ -79,6 +79,24 @@ def __init__(self, cjson, molecule_id):
         self._id = molecule_id
         self._svg_ = None
 
+    @property
+    def cjson(self):
+        if self._cjson_ is None:
+            # Try to update the cjson
+            try:
+                resp = GirderClient().get('molecules/%s/cjson' % self._id)
+            except HttpError as e:
+                fail_str = 'Molecule does not have 3D coordinates.'
+                if fail_str in e.responseText:
```

Comment:
I'm not a huge fan of using the error text. May be the `molecules/%s/cjson` endpoint should return an empty object or `None` if there are no 3D coordinates?

Reply:
Okay. Yeah I think we could do that. The relevant code block is [here](https://github.com/OpenChemistry/mongochemserver/blob/8d5873ad8290e7cf345aff8cb43b49fec3342ee9/girder/molecules/molecules/molecule.py#L388-L392).

Reply:
Fixed in openchemistry/mongochemserver#173 and the latest commit here.